workspaces/copilot/plugins/copilot-backend/src/client/GithubClient.ts

```diff
@@ -100,3 +115,74 @@ export class GithubClient implements GithubApi {
     return response.json() as Promise<T>;
   }
 }
+
+/**
+ * @internal
+ *
+ * This is just a bridge function to make #2828 work on short notice, long term, we should rely on CopilotMetrics instead of Metric
```

Comment:
```suggestion
 * This is just a bridge function as the Usage API endpoint has been removed and this will make it work on short notice, long term, we should rely on CopilotMetrics instead of Metric
```